tools/build_create_carrier_curl_requests/build_curls.py

```diff
@@ -24,7 +24,9 @@
 CANADAPOST_CUSTOM_WORKFLOW_CARRIERS = [
     "CanadaPostAccount",
 ]
-
+AMAZON_CUSTOM_WORKFLOW_CARRIERS = [
```

Comment:
suggestion: let's call this something like this instead? Will help future expansions of this list
```suggestion
OAUTH_CUSTOM_WORKFLOW_CARRIERS = [
```